db/version_set.h

```diff
@@ -923,7 +923,8 @@ class VersionSet {
 
   // printf contents (for debugging)
```

Comment:
`// If sst_file_number is > 0, only prints manifest info for specified SST file number`